lib/services/dynamodb.js

```diff
@@ -47,7 +47,18 @@ AWS.util.update(AWS.DynamoDB.prototype, {
    * @api private
    */
   retryDelays: function retryDelays(retryCount) {
-    var delay = retryCount > 0 ? (50 * Math.pow(2, retryCount - 1)) : 0;
+    var retryDelayOptions = this.config.retryDelayOptions;
+
+    if (typeof retryDelayOptions.base !== 'number') {
+        retryDelayOptions.base = 50; // default for dynamodb
+    }
+
+    var delay;
+    if (typeof retryDelayOptions.customBackoff === 'function') {
+      delay = AWS.util.calculateRetryDelay(retryCount, retryDelayOptions);
+    } else {
+      delay = retryCount > 0 ? AWS.util.calculateRetryDelay(Math.max(0, retryCount - 1), retryDelayOptions) : 0;
```

Comment:
I think you're right based on the documentation on DynamoDB's page. Today the SDK will perform the first retry immediately, but happy to change that (and simplify the code!)